ModelBase/src/model/TreeManager.h

```diff
@@ -490,6 +506,9 @@ inline Node* TreeManager::root(){ return root_; }
 inline QString TreeManager::name() { return name_; }
 inline void TreeManager::setName(const QString& name) { name_ = name; }
 
+inline QString TreeManager::managerName() { return managerName_; }
```

Comment:
I don't like the name of this field. Now manager has the `TreeManager::name` and `TreeManager::managerName` fields. These are totally identical, unless one reads the comments. We should call it something else. How about `revisionName` or is there an even more precise name?